### ISSUE TITLE
Feature request: Windows drive letter conversion

### ISSUE THREAD

**Issue by myuseringithub** [anodynos/upath#27]
Hi, Is there support for converting absolute Windows paths including the drive letter to Unix path ? 
i.e. `C:\` to `/C/`

Thanks

**Comment by anodynos** [comment]
No there is isn't - if you see the docs/tests, the `:` is retained as it should be, it wont work on windows otherwise.

But it's a very easy replacement you can do on the user space, just a regexp :-)

**Comment by myuseringithub** [comment]
That's what I did, just thought it is related to the module and could be exposed as an option in the module api.

**Comment by anodynos** [comment]
Thanks @myuseringithub, but will the `/C/` continue to work on Windows? I guess not... 
I'm sorry I gave up Windows for the past ~5 years to Linux, never going back for dev at least :-)

**Comment by myuseringithub** [comment]
@anodynos Well, Windows is quite different since 6 years. Many nice features superior to what the different Linux OS provide, virtual desktops is my favorite. 
I develop in Docker containers so that's not an issue.

**Comment by anodynos** [comment]
Nah, not going back to Windows, I hate it. 
My Kubuntu Linux box works so great, bless it! I use it as my main desktop PC as well, watch movies, browse the net, even play some games with WINE (they run as good if not faster than Winblows). 

And the greatest feature of all? I have it installed on a fast External SSD (Samsung T1) and I boot with it on my 3 home PCs & laptops, my work's laptop & even I tried with my father's 10 year's 2GB RAM old laptop and IT WORKS (camera, Bluetooth, GPU, audio, everything). Linux rocks these days!

No more reinstalling, reconfiguring, re-everything, just cause I changed hardware!
Paying for extra licenses is not my only concern, but saving days or weeks of work, it absolutely is!
